x-pack/plugins/ml/public/application/contexts/kibana/kibana_context.ts

```diff
@@ -11,10 +11,12 @@ import {
   KibanaReactContextValue,
 } from '../../../../../../../src/plugins/kibana_react/public';
 import { SecurityPluginSetup } from '../../../../../security/public';
+import { LicenseManagementUIPluginSetup } from '../../../../../license_management/public';
 
 interface StartPlugins {
   data: DataPublicPluginStart;
   security: SecurityPluginSetup;
+  licenseManagement: LicenseManagementUIPluginSetup;
```

Comment:
As it is optional we need to add the `?`

```js
licenseManagement?: LicenseManagementUIPluginSetup;
```

Reply:
Changes in 73e832b65c118e802e2e5bb1a85d834bee95f5af